ArchiSteamFarm/Steam/Data/AccessTokenResponse.cs

```diff
@@ -0,0 +1,22 @@
+
+using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json;
+
+namespace ArchiSteamFarm.Steam.Data {
+	[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
+	public class AccessTokenResponse {
+		[JsonProperty(PropertyName = "success", Required = Required.Always)]
+		internal bool Success { get; private set; }
```

Comment:
`Success` is not a boolean but `EResult`. Remove this and make the class inherit from `EResultResponse` instead.